src/common/options.cc

```diff
@@ -2102,8 +2102,13 @@ std::vector<Option> get_global_options() {
     .set_description(""),
 
     Option("osd_max_write_size", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
+    .set_min(4)
     .set_default(90)
-    .set_description(""),
+    .set_description("Maximum size of a RADOS write operation in megabytes")
+    .set_long_description("This setting prevents clients from doing "
+        "very large writes to RADOS.  If you set this to a value "
+        "below what clients expect, they will receive an error "
+        "when attempting to write to the cluster."),
```

Comment:
Given the failure you're attempting to avoid, this should mention or use the ```related_to``` option about mds_dir_max_commit_size as well?

Reply:
Actually it was the mds log segment size -- not sure we can realistically add related_to hooks for all the related client side settings, as it also includes any layouts